x-pack/plugins/watcher/public/application/sections/watch_edit_page/components/json_watch_edit/simulate_watch_results_flyout.tsx

```diff
@@ -42,19 +43,41 @@ export const SimulateWatchResultsFlyout = ({
 
   const { actionModes } = executeDetails;
 
+  const conditionNotMetActionStatus = (mode: string) => {
+    switch (mode) {
+      case 'simulate':
+      case 'force_simulate':
+        return 'not simulated';
```

Comment:
I think these strings are then displayed directly in the UI, right? If so, could you please add the translation to them? You can use `i18n.translate` function for that.